orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/oracle/OracleImageFinder.groovy

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Please avoid adding more Groovy classes in main source tree whenever possible.

Reply:
Is Java preferred?

Reply:
Yes